frontend/fingertips-frontend/lib/registry.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { expect } from '@jest/globals';
+import { render } from '@testing-library/react';
+import StyledComponentsRegistry from './registry';
+import { isBrowser } from './utils';
+import styled from 'styled-components';
+
+jest.mock('./utils');
+
+const mockIsBrowser = isBrowser as jest.MockedFunction<typeof isBrowser>;
+
+const StyledDiv = styled('div')<{ bgColor: string }>(({ bgColor, color }) => ({
+  backgroundColor: bgColor,
+  color,
+}));
+
+const SomeComponent = () => {
+  return (
+    <StyledDiv bgColor="blue" color="red">
+      Hello World
+    </StyledDiv>
+  );
+};
+
+describe('StyledComponentsRegistry', () => {
+  it('should return inline styles when rendering the wrapped component client side', () => {
+    mockIsBrowser.mockReturnValue(true);
```

Comment:
I think you could do `jest.mocked(isBrowser).mockReturnValue(true)` here, to avoid having to create `mockIsBrowser`.

Reply:
Nice! updated

Reply:
Took me a while to find `jest.mocked`, but I'm a big fan!